RELEASE.md

```diff
@@ -8,11 +8,17 @@
 
 ## Migration guide from Kedro 0.18.* to 0.19.*
 
+## Major features and improvements
+
+# Release 0.18.2
+
 # Upcoming Release 0.18.2
 
 ## Major features and improvements
 
 ## Bug fixes and other changes
+
+* Bumped `pyyaml` upper-bound to 7.0.0. Version 6.0.0 of PyYAML dropoed Python 2.7 support and relaxing this makes Kedro compatible with the [pyodide](https://pyodide.org/en/stable/usage/loading-packages.html#micropip) stack.
```

Comment:
```suggestion
* Bumped `pyyaml` upper-bound to make Kedro compatible with the [pyodide](https://pyodide.org/en/stable/usage/loading-packages.html#micropip) stack.
```
Just to keep the release notes snappy 😀 I don't think anyone cares about Python 2.7 support. Probably most people would think it's a typo for 3.7...